doc/self-hosting.md

```diff
@@ -130,3 +130,17 @@ rotation logic.
 
 [github rate limit]: https://developer.github.com/v3/#rate-limiting
 [personal access tokens]: https://github.com/settings/tokens
+
+
+Separate frontend hosting
+-------------------------
+
+If you want to host the frontend on a separate server, such as cloud storage
+or a CDN, you can do that. Just copy the built `index.html` there.
+
+To help out users, you can make the Shields server redirect the server root.
+Set the `FRONTEND_REDIRECT_URI` environment variable:
+
+```sh
+FRONTEND_REDIRECT_URI=http://my-custom-shields.s3.amazonaws.com/
```

Comment:
In this file we have `FRONTEND_REDIRECT_URI`, but in `lib/server-config.js`, `shields.example.env` and https://github.com/badges/ServerScript/pull/1 there is `FRONTEND_REDIRECT_URL` (URL instead of URI).

Reply:
Good catch, thank you!